wizard/models.py

```diff
@@ -247,7 +247,8 @@ def clean(self):
         cols = columns_count_first_line(self.raw_content)
 
         if not columns_all_the_same_count(self.raw_content, cols):
-            raise InvalidAutomlFormatException("Number of cols non coherent in %s" % self.raw_content)
+            pass
+            #raise InvalidAutomlFormatException("Number of cols non coherent in %s" % self.raw_content)
```

Comment:
Yeah, one the the default dataset don't pass this test.
So I don't know if it's the dataset witch have a problem or if this test is useless (event if it was on a TODO comment)